tests/core-addon/IonCore.test.js

```diff
@@ -427,7 +427,7 @@ describe('IonCore', function () {
   describe('_updateInstalledStudies()', function () {
     it('adds the ionInstalled property', async function () {
       // We don't expect any update task to be running now.
-      assert.equal(this.ionCore._updateInstalledTask, null);
+      assert.equal(this.ionCore._updateInstalledTask, undefined);
```

Comment:
AH! This is vestigial code, it's testing something that's no longer there :)
Would you kindly remove this line and the comment immediately above?

Reply:
Removed.

Reply:
Awesome, thank you so much!